exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestCTAS.java

```diff
@@ -357,6 +358,62 @@ public void testTableIsFoundWithinWorkspaceWhenNameStartsWithSlash() throws Exce
     }
   }
 
+  @Test
+  public void testCTASWithTrueMapInSelect() throws Exception {
```

Comment:
I believe @arina-ielchiieva's [comment](https://github.com/apache/drill/pull/1844#discussion_r315588844) is also applicable to this situation. In addition, please replace ```TrueMap``` with ```Dict``` in test method names.